recipes/_omnibus_toolchain.rb

```diff
@@ -21,31 +21,41 @@
 toolchain_name = node['omnibus']['toolchain_name']
 toolchain_version = node['omnibus']['toolchain_version']
 
-if solaris_10?
-  # create a nocheck file for automated install
-  file '/var/sadm/install/admin/auto-install' do
-    content <<-EOH.flush
-      mail=
-      instance=overwrite
-      partial=nocheck
-      runlevel=nocheck
-      idepend=nocheck
-      space=ask
-      setuid=nocheck
-      conflict=nocheck
-      action=nocheck
-      basedir=default
-    EOH
-    owner 'root'
-    group 'root'
-    mode '0444'
-  end
+if File.exist?('/opt/omnibus-toolchain') || File.exist?('/opt/angry-omnibus-toolchain')
+  Chef::Log.warn('Assuming the existence of /opt/(angry-)omnibus-toolchain means that the package is already installed.')
+else
+  if solaris_10?
+    # create a nocheck file for automated install
+    file '/var/sadm/install/admin/auto-install' do
+      content <<-EOH.flush
+        mail=
+        instance=overwrite
+        partial=nocheck
+        runlevel=nocheck
+        idepend=nocheck
+        space=ask
+        setuid=nocheck
+        conflict=nocheckit
+        action=nocheck
+        basedir=default
+      EOH
+      owner 'root'
+      group 'root'
+      mode '0444'
+    end
+
+    case node['kernel']['machine']
+    when 'i86pc'
+      package_url = "https://chef-releng.s3.amazonaws.com/omnibus/omnibus-toolchain/#{toolchain_name}-#{toolchain_version}-1.i86pc.solaris"
+    when 'sun4v', 'sun4u'
+      package_url = "https://chef-releng.s3.amazonaws.com/omnibus/omnibus-toolchain/#{toolchain_name}-#{toolchain_version}-1.sun4v.solaris"
+    end
 
-  case node['kernel']['machine']
-  when 'i86pc'
-    package_url = "https://chef-releng.s3.amazonaws.com/omnibus/omnibus-toolchain/#{toolchain_name}-#{toolchain_version}-1.i86pc.solaris"
-  when 'sun4v', 'sun4u'
-    package_url = "https://chef-releng.s3.amazonaws.com/omnibus/omnibus-toolchain/#{toolchain_name}-#{toolchain_version}-1.sun4v.solaris"
+    install_options = '-a auto-install'
+  elsif aix?
+    package_url = "http://chef-releng.s3.amazonaws.com/omnibus/omnibus-toolchain/#{toolchain_name}-#{toolchain_version}-1.powerpc.bff"
```

Comment:
You should define a value for `install_options` globally in this recipe or in this branch of code.